src/vmm/src/rpc_interface.rs

```diff
@@ -1827,60 +1839,6 @@ mod tests {
         );
     }
 
-    #[test]
-    fn test_build_microvm_from_requests() {
```

Comment:
can we modify the test to adapt to the new changes instead of removing it completely?

Reply:
I can't see a way to do it, the only way would be to just test a function further up that then calls this (which we are already doing). I think originally this design existed for this test, but as coverage shows this actually adds more uncovered lines.